apps/crypto/src/App.jsx

```diff
@@ -1,6 +1,39 @@
 import React, { useState } from "react";
 import styled from "styled-components";
 import { isTriggered } from "@felvin-search/core";
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  LineElement,
+  Title,
+  Tooltip,
+  Legend,
+} from "chart.js";
+import { Line } from "react-chartjs-2";
+import faker from "faker";
+
+ChartJS.register(
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  LineElement,
+  Title,
+  Tooltip,
+  Legend
+);
+
+export const options = {
+  responsive: true,
```

Comment:
Although the chart is surely responsive, it's not doing much help for mobile users as can be seen below. What do you think? 

![image](https://user-images.githubusercontent.com/60931421/145706732-cbde9421-f272-494b-b0ec-4fb0342233aa.png)

Reply:
hmmm, that's interesting. Thought it would keep the format for mobile. Gotta go through the chart.js to see if they have any options for this